CRM/ManualDirectDebit/Batch/BatchHandler.php

```diff
@@ -315,8 +314,8 @@ private function submitDDPayments() {
    * @param string $IDs list of IDs separated by comma
```

Comment:
Please update docblock and parameter name to reflect current usage (single integer instead of comma separated id's).